pmoves/services/hi-rag-gateway-v2/embeddings.py

```
@@ -0,0 +1,62 @@
+
+    logger.info("Embedding providers unavailable; falling back to %s", MODEL)
+    try:
+        emb = _get_embedder().encode([text], normalize_embeddings=True)
+        if hasattr(emb, "tolist"):
+            emb = emb.tolist()
+        candidate = emb[0] if isinstance(emb, list) and emb else None
+        candidate = _coerce_vector(candidate)
+        if not candidate:
+            raise RuntimeError("sentence-transformers returned empty vector")
+        return candidate
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)